src/Query/index.ts

```diff
@@ -108,22 +108,30 @@ export class Query {
     const parts: Part[] = [];
     const searchPropertyMapping = getPropertyMapping();
 
+    // DOP-3934 - test without having this hardcoded matches
     // if we need to boost for matching slug on an exact rawQuery match
-    const boostedStrings = strippedMapping[this.rawQuery.trim()];
-    if (Array.isArray(boostedStrings) && typeof boostedStrings[0] === 'string') {
-      parts.push({
-        text: {
-          path: 'strippedSlug',
-          query: boostedStrings,
-          score: { boost: { value: 100 } },
-        },
-      });
-    }
+    // const boostedStrings = strippedMapping[this.rawQuery.trim()];
```

Comment:
My understanding is that the mapping includes terms that don't have `$` in them, meaning those terms won't be boosted anymore. I didn't see this in the A/C of the original ticket, so I'm just wondering if this is okay to do?

Reply:
good point. i think we should just leave the mapping (this was for testing in qa) and see if there is an overall improvement with one change at a time - changing the mapping, and separately removing this hardcoded mapping